llama_stack/providers/registry/safety.py

```diff
@@ -21,6 +21,7 @@ def available_providers() -> List[ProviderSpec]:
             api=Api.safety,
             provider_type="inline::prompt-guard",
             pip_packages=[
+                "accelerate",
```

Comment:
would `transformers[accelerate]` work here? (if so, I think it's better to fetch these dependencies through the optional-dependencies for the package that we actually import ourselves)

Reply:
Yup it does, updated, thanks.